cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -36,6 +36,22 @@ parameter "param_kubecost_host" do
   # No default value, user input required
 end
 
+parameter "param_allow_shared_core" do
+  type "string"
+  category "Allow Shared Core"
+  label "Shared core"
+  allowed_values "true", "false"
+  description "Enable or disable the use of shared cores in calculating costs and resource savings in Kubecost"
```

Comment:
@joseluiszuflores could we re-phrase the description to something like: "Whether to include shared core node types into recommendation."

Reply:
yeah we have modified  this one by  ***Whether to include shared core node types into recommendation.***